styles/16colors.json

```diff
@@ -0,0 +1,204 @@
+{
+  "document": {
+    "block_prefix": "\n",
+    "block_suffix": "\n",
+    "color": "7",
+    "margin": 2
+  },
+  "block_quote": {
+    "indent": 0,
+    "color": "3",
+    "indent_token": "│ ",
+    "italic": true,
+    "margin": 1
+  },
+  "paragraph": {},
+  "list": {
+    "level_indent": 2,
+    "color": "7"
+  },
+  "heading": {
+    "block_suffix": "\n",
+    "color": "4",
+    "bold": true
+  },
+  "h1": {
+    "prefix": " ",
+    "suffix": " ",
+    "color": "7",
+    "background_color": "4",
+    "bold": true
+  },
+  "h2": {
+    "prefix": "## "
+  },
+  "h3": {
+    "prefix": "### "
+  },
+  "h4": {
+    "prefix": "#### "
+  },
+  "h5": {
+    "prefix": "##### "
+  },
+  "h6": {
+    "prefix": "###### ",
+    "color": "5",
+    "bold": false
+  },
+  "text": {},
+  "strikethrough": {
+    "crossed_out": true
+  },
+  "emph": {
+    "italic": true,
+    "color": "3"
+  },
+  "strong": {
+    "bold": true,
+    "color": "3"
+  },
+  "hr": {
+    "color": "0",
+    "format": "\n―――――――――――――――――――\n"
+  },
+  "item": {
+    "block_prefix": "• "
+  },
+  "enumeration": {
+    "block_prefix": ". ",
+    "color": "6"
+  },
+  "task": {
+    "ticked": "[✓] ",
+    "unticked": "[ ] "
+  },
+  "link": {
+    "color": "6",
+    "underline": true
+  },
+  "link_text": {
+    "color": "5",
```

Comment:
Preference to have the same color as link_text, it makes it easier to understand they are the same units.